csm/conf/configure.py

```diff
@@ -227,19 +231,19 @@ async def _set_unsupported_feature_info(self):
         """
 
         def get_component_list_from_features_endpoints():
-            Log.info("Get Component List.")
+            Log.info("Config: Getting component list.")
             feature_endpoints = Json(
                 const.FEATURE_ENDPOINT_MAPPING_SCHEMA).load()
             component_list = [feature for v in feature_endpoints.values() for
                               feature in v.get(const.DEPENDENT_ON)]
             return list(set(component_list))
         try:
-            Log.info("Set unsupported feature list to ES.")
+            Log.info("Config: Setting unsupported feature list to ES.")
```

Comment:
ES is not in picture now. Anyways comment this code

Reply:
please change ES to storage/database and yes comment this code as asked by Pawan

Reply:
updated